### ISSUE TITLE
T&T Dispute resolution call - Friday July 28th at 10 am EST

### ISSUE THREAD

**Issue by raghavrmadya** [filecoin-project/notary-governance#949]
**Agenda**

Public discussion and vote on open disputes.

**Time and Location**

July 28th 2023 - 10 am EST

Location - [Zoom]([https://protocol.zoom.us/j/95289865541](https://www.google.com/url?q=https://protocol.zoom.us/j/95289865541&sa=D&source=calendar&ust=1690734674661544&usg=AOvVaw1eSNeGbrgozj-sc2RsCAOS)

**Proposed flow for the call** (old)

- The 10 disputes listed in this issue will be discussed on this call. 
- 5 minutes shall be spent on each issue with a hard stop on each dispute. 
- 4 minutes will be allocated to a discussion of the disputes and 1 minute to a public vote. All participants on the call are able to vote using the Zoom vote feature. 
- The outcome of the vote will be shared after the vote and the T&T Lead will take the necessary steps to execute the community decision.

**Proposed flow for the call addressing possibility of sybil** 
Additionally, there have been valid concerns about a potential sybil during the voting period. To address this concern, I would like to propose we leverage quadratic vote using this [tool](https://quadraticvote.co/) as described [here](https://docs.snapshot.org/user-guides/proposals/voting-types) 

The proposed voting strategy is as follows:

 The first 10 disputes listed in this issue will be discussed on this call.
- 5 minutes shall be spent on each issue with a hard stop on each dispute. 
- 4 minutes will be allocated to a discussion of the disputes and 1 minute to a public vote (described below)
- The outcome of the vote will be shared after the vote and the T&T Lead will take the necessary steps to execute the community decision.

**Voting strategy**

1. Everyone that joins the call receives 10 credits. 
2. During the voting period, members can use their credits to indicate the strength of their preference for the proposal as - agree, disagree, and abstain
3. Once a member has used up their credits, they will not be able to vote further but can partake during the discussion period. 

If you would like to add a specific issue to the agenda, please request time by adding a brief description of the issue/proposal/topic in the comments of this issue. Please have your request submitted in this issue at least 48hrs before the proposed meeting to ensure it is included in the agenda.

Please link the issue in the comments below and it will be added to the agenda.

**Comment by woshidama323** [comment]
Thanks for opening such an issue then we can fix it ASAP
We want #913 to be listed, We have answered all the questions VPN etc, and wish this issue could be closed ASAP 
And it has impacted many other LDNs which list TopBlocks in their LDN  for example [2049](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2049) [2054](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2054) etc.

**Comment by woshidama323** [comment]
And also about #934, Many notaries have been involved in this issue including us Topblocks
Please also help to add it in the list

**Comment by kernelogic** [comment]
Add #947 please

**Comment by nicelove666** [comment]
In last night's meeting we briefly discussed , we would like to add #917, thanks

**Comment by xiaoming123222** [comment]
There has been no response to question #937, and I would like to add it to this meeting. Thank you.

**Comment by dahuajishu** [comment]
Regarding #914, we did not speak and it was closed. Could you reopen it or join the discussion at this meeting?

**Comment by c12xf** [comment]
> Regarding #914, we did not speak and it was closed. Could you reopen it or join the discussion at this meeting?

Thank you for your reminder. I think it's necessary.

**Comment by herrehesse** [comment]
We encourage everyone to openly discuss their disputes, and all individuals are welcome to share their perspectives. However, it is essential that all participants adhere to presenting facts and evidence-based arguments. Opinions about dispute cases will not be considered as substantial as evidence-based facts.

Our aim is to efficiently resolve disputes, and we are committed to working towards a swift resolution for each case.

**Comment by twosson** [comment]
Regarding #2058. I hope to be able to continue this process. Our company is engaged in AI technology. The stored video data includes part of the verification data. I guess this should be allowed?

**Comment by twosson** [comment]
Regarding #2058 . I hope to be able to continue this process. Our company is engaged in AI technology. The stored video data includes part of the verification data. I guess this should be allowed?

**Comment by Aaron01230** [comment]
add #934 pls.

**Comment by Destore2023** [comment]
#934 would like to be considered in the list. @raghavrmadya Would you please specify the content of the list ( "The 10 disputes listed in this issue will be discussed on this call.")

**Comment by Joss-Hua** [comment]
How to know if a LDN is in the disputes?

**Comment by Joss-Hua** [comment]
#1725 #1726 #1727 #1728 #17259

**Comment by 1ane-1** [comment]
#934

**Comment by nj-steve** [comment]
Hello #898 please

**Comment by herrehesse** [comment]
Hi @raghavrmadya - #905 please

**Comment by raghavrmadya** [comment]
Thanks everyone. As mentioned in the proposal, all issues received before 10 am EST today (48 hrs before the call) will be discussed in the upcoming call. All other issues will be tabled for the next call. 

A kind reminder that this is an experiment to have a decentralized dispute resolution process

**Comment by raghavrmadya** [comment]
I've also edited the original proposal by adding the voting strategy section without deleting anything that was initially proposed.

**Comment by kernelogic** [comment]
How many issues going to make to the call? Everybody gets 10 credits to vote at most 10 issues?

**Comment by raghavrmadya** [comment]
Comments requesting discussion on LDNs were not taken into consideration as this call is proposed to be regarding disputes on notary governance

**Comment by herrehesse** [comment]
In a standard judicial process, both the accused and the accuser present their arguments and evidence to an impartial jury, composed of individuals not directly involved in the case. Following their deliberation, the case is then brought before a judge, who considers all the information presented before making a final decision.

In a concerning departure from the standard judicial process, the current situation involves a plan to conduct a vote on matters related to abusive groups of notaries and throwaway GitHub accounts (clients). This vote would be carried out by the very entities facing allegations, without the involvement of an impartial jury or judge. Furthermore, these entities are forming alliances against the accusers, raising concerns about most likely bias and unfair outcomes.

This approach lacks logic and would lead to heavily skewed results. As a significant part of the "governance" team, it is crucial to take charge and make difficult decisions, even if it means facing adversaries. Fair judgments must be based on clear and compelling evidence, not cheaply spoken words. Allowing the accused to vote on their own potential wrongdoings, especially when substantial evidence exists, is beyond my capabilities of understanding.

To govern the community responsibly and impartially, credible evidence should be the foundation of decisions. If the accused can only defend themselves with words and not substantial evidence, decisive judgments must still be made for the greater good of the community. Emotions should not dictate the process to ensure just outcomes.

If making clear or fair judgments becomes challenging, prompt action should be taken to hire a third-party entity capable of impartial decision-making. In cases where evidence of wrongdoing is evident, there should be no room for voting on whether a violation occurred; action should be taken based on the presented evidence, just as nobody votes on whether they ran a red light with their vehicle.

Two proposed approaches to address the current issues are as follows:

- Connect 1-3 notaries directly to LDNs, holding them fully responsible for data quality, onboarding, distribution, and retrieval results in contact with the client. This approach would require them to conduct thorough due diligence on the LDNs they sign. If significant issues arise, as we can see happen across the board today, they would be held accountable and subject to removal as notaries. Enforcing such responsibility would lead to better governance of Filecoin+.

- Emphasise the urgency of implementing V5 as soon as possible. Instead of voting on matters while abusers continue their actions unchecked, a new V5 round should be initiated. 

In conclusion, allowing groups of accused notaries and clients to vote on their own accusations, based on factual evidence from the blockchain, is an alarming and detrimental idea. Decisive action, the involvement of an impartial third party if necessary, and the implementation of the proposed approaches can lead to a more robust and fair ecosystem.

**Comment by raghavrmadya** [comment]
@herrehesse, thanks for the feedback. Your argument is based on the foundation that the standard judicial process works well. I disagree. Having a single judge has its own inefficiencies and unfortunately in this governance model, we don't have a supreme court. Having a single judge also concentrates power let alone the risk of threats and physical harm to such an individual. 

From my own research and reading on quadratic voting, the strategy is very much designed to address the concerns you list in the 3rd paragraph of your comment above.

Let's work with an example here - 

1. 50 people show up to the call on Friday and each get 10 credits before we start discussing the open disputes. 

2. We discuss dispute #x (4 minutes) 

3. The individual/entity under dispute use all their credits to vote in their own favor. Others who join the call choose to vote with their credits either in favor, against, or abstain (credits are to be used to vote abstain)

4. We move to discuss dispute #y (4 minutes)

5. Those that have used the credits don't get a fresh batch of 10 credits. They must choose which issues they want to vote on.  

That said, we can collectively define the constraints of this voting strategy. Some ideas include:

1. Mandating that each member on the call must use all credits across the 8 issues. 
2. Capping the participants to those who join in the first 2 minutes to ensure people who join later do not get to sway the results.

**Comment by herrehesse** [comment]
I am not voting on accusations based on factual evidence @raghavrmadya. You either crossed the red light or not.

**Comment by herrehesse** [comment]
@raghavrmadya all of the accused use all of their “tokens” to vote against all of their own disputes.

**Comment by DSS-AL** [comment]
If FIL+ is to continue (there remains a question mark around whether or not it should), then DSS believe there needs to be a structural change to the governance procedure.

In lieu of any alternative, DSS support this proposal. 

As frustrating as it may be if an application must pass through even greater scrutiny to ensure the integrity of an application and ongoing retrievability of the data, we support this position.

One condition that DSS would require is a rotation of members with a defined time period, suggest a 3 months maximum tenure for the members of this panel.

**Comment by DSS-AL** [comment]
**Removed last comment as it was posted in the wrong discussion.**

Surely the purpose of having the T&T group is to act as a point of escalation to assess and resolve any disputes?

The proposed process allows RELATED parties to vote potentially maliciously in favour and it also allows for UNRELATED parties to vote potentially maliciously against.

I would consider a fair and reasonable process of resolution to be much like the E-FIL+ process including KYC of the applicant, KYC of the SPs storing the data, and a review of sample data, etc

The current process is deeply flawed.

**Comment by Joss-Hua** [comment]
> If FIL+ is to continue (there remains a question mark around whether or not it should), then DSS believe there needs to be a structural change to the governance procedure.
> 
> In lieu of any alternative, DSS support this proposal.
> 
> As frustrating as it may be if an application must pass through even greater scrutiny to ensure the integrity of an application and ongoing retrievability of the data, we support this position.
> 
> One condition that DSS would require is a rotation of members with a defined time period, suggest a 3 month maximum tenure for the members of this panel.

The learning threshold for filecoin+ is very high, and for many notaries, it takes three months to understand what they need to do and how to do it. The rotation may block out all new members.

**Comment by Aaron01230** [comment]
It is very reasonable for the notary to vote on the proposal of FIL+.
In addition, 18 notaries are involved in 934, and 5 minutes of discussion time is not enough.

**Comment by raghavrmadya** [comment]
As there is some pushback on if this approach of voting on T&T is conducive or not, I would like other members in the community, including members of the governance team and notaries to please provide their input on the pros and cons of proceeding with this call tomorrow

**Comment by kernelogic** [comment]
I think we all know the turn out of this vote is going to be in favor of the "defendants" because they are the majority here. 

My question is, will the governance team accept and execute whatever the turn out to be?

**Comment by galen-mcandrew** [comment]
I really appreciate the amount of professionalism and discourse happening in this thread. I think it shows the power and potential of our open governance and decision making. That said, I have some concerns about pushing forward this specific process for tomorrow’s (July 28) call. My concerns are primarily around: Proof of Concept testing, High stakes topics, and Equitability. I’ll expand on these, and then make alternative proposals.

**PoC Testing:** From reading the thread, it sounds like the process (quadratic voting through this [tool](https://quadraticvote.co/)) was proposed mid-week. Additionally, I’m not sure we have used any voting mechanism before across the Fil+ working groups. I think this quadratic mechanism could work, but we have not had time to run a Proof of Concept test, and there are many unanswered questions even about the tool itself: do users need accounts, how will we handle sybils, how transparent are votes and outcomes, will the platform work in different regions? 

**High Stakes:** There is a case to be made for just trying it live Friday and learning from rapid testing, which I usually support. That said, the specific disputes up for discussion on the call are not simple, concise, or inconsequential. These involve clients, notaries, storage providers, and new programmatic proposals. I haven’t even had the time to tally up the potential impact here, in number of stakeholders, volume of DataCap, or potential programmatic costs (new developer tooling to support the proposed automations). Thankfully, our program has successfully iterated and improved as new tooling, insights, stakeholders, and market patterns emerge, rather than being mired in precedent. That said, I think the costs here outweigh the potential for rapid bias to action. 

**Equitability:** Despite a few comments in the community, the Filecoin Plus program and Governance team remain devoted to an open, transparent, and equitable design philosophy. We operate in a global marketplace, which makes synchronous decision-making extremely difficult. I fully relate to the frustrations felt by the community that decision-making and resolution takes too long, but the answer is not synchronous live meetings. Our community is made up of professional adults, with packed schedules and full lives (I’m drafting this at midnight hoping my toddler doesn’t ask for yet another glass of water). Our processes need to account for timezones, languages, and competing professional demands, so that our stakeholders are able to read, research, and weigh-in on the issues that concern them. Slow is smooth, and smooth is fast.


Given these concerns, I propose we do not use a voting mechanism to determine the outcomes of the open disputes on the docket tomorrow. I suggest drafting a sample dispute, where someone can agree to play the role of the accused party, and running a test of this quadratic voting tool on the call. That will at least allow the present stakeholders to stress test the tool and provide feedback. If we decide to move forward with voting, we will need to draft a more robust governance proposal for a process; which defines what types of disputes would qualify, how we’ll handle threats (like sybils), what the timelines would be, and how it will remain equitable. To handle the seven issues on the docket, I propose the following process:

- 1 minute overview of dispute by T&T lead - summary of case (type of dispute, existing evidence/counter-evidence, proposed resolution)
- 1 minute defense and counter-evidence by accused (or representative) - can be admission or denial, must remain professional and evidence-based
- 2 minute open questions by group - asking for clarifications or more details from either accuser or accused, not for voicing opinions 
- 1 minute review and potential ruling - if overwhelming evidence and insufficient defense or justification then resolve, otherwise dispute will remain open for additional investigation 


I understand, respect, and appreciate the fervor of this community in pushing the Filecoin network and the Fil+ program forwards. This is our professional livelihood, which is why it deserves professional consideration. The Governance team is very hard at work to keep iterating and resolving new challenges, and we will continue to design in the open and include as many stakeholders as possible. This means constantly working to balance competing priorities and seek compromise. I look forward to hearing constructive feedback, and seeing how we refine the dispute resolution process to best serve our ecosystem.

**Comment by Filplus-govteam** [comment]
Based on the comments above, we will not vote on disputes as there isn't a majority consensus. I propose that we follow the proposed process above as this is the most comprehensive proposal I have seen on this thread so far. If others have a proposal between now and 9 am EST, kindly share.

**Comment by KennethAd** [comment]
What's the point of voting then?

**Comment by raghavrmadya** [comment]
Based on the comments on this thread, we don't have enough consensus to hold a vote. I would like to propose that we still hold a discussion on the aforementioned issues.

**Comment by raghavrmadya** [comment]
0xXPunkX comment violated the [code of conduct](https://medium.com/filecoin-plus/fil-code-of-conduct-9cd044e7bcaf?source=collection_home---4------2-----------------------)

**Comment by raghavrmadya** [comment]
<img width="717" alt="Screenshot 2023-07-27 at 11 39 39 PM" src="https://github.com/filecoin-project/notary-governance/assets/91579519/bb615f8b-b3be-417b-ae22-7ea696e2f5a3">

**Comment by stcouldlisa** [comment]
If there are still many uncertainties, then we can also set the meeting as tomorrow. Compared with the meeting time, it may be more meaningful for everyone to reach a consensus on the meeting.

**Comment by Aaron01230** [comment]
Because these issues involve a relatively large proportion of notaries in China, please set up a Chinese translation and a Chinese jury for this discussion

**Comment by Destore2023** [comment]
> Because these issues involve a relatively large proportion of notaries in China, please set up a Chinese translation and a Chinese juror for this discussion, forgive my english is poor

@raghavrmadya  I agree with this. language barriers prevent those controversial persons from arguing within such a  limited time. a translator may help them to do so and it's fair in this meeting.

**Comment by raghavrmadya** [comment]
@Aaron01230 @Destore2023 , thanks for raising this point. I agree and I will try my best to find a translator before the call today.

**Comment by raghavrmadya** [comment]
As shared on the T&T WG call earlier this week, the initial intent of this call was to make up for the missed calls in the last 2 weeks due to holidays and team offsites. As we did not discuss any specific disputes in the call earlier this week, this call will focus solely on discussing the following 7 issues:

**Comment by raghavrmadya** [comment]
913
934
917
947
937
898
905

**Comment by raghavrmadya** [comment]
The recording from this call is available here - https://protocol.zoom.us/rec/share/b3E1tci4dcuprXXT2QnEXbwNychvObfYs43hKDeWkUNpim08BqMJUqRMUgVE7AY.1nZV-WJ6BECtul8E 
Passcode: hay=zJ2u
